src/name_bazaar/ui/pages/offering_create_page.cljs

```diff
@@ -305,7 +306,9 @@
               :offering (transaction-data->form-data offering)}])]]))))
 
 (defmethod page :route.offerings/create []
-  [app-layout
-   [ui/Segment
-    [:h1.ui.header.padded "Create Offering"]
-    [offering-form]]])
+  [:div.wrapper
+   [name-bazaar-meta-tags]
```

Comment:
:title "NameBazaar - Create Offering"
:description "Create an offering to sell your ENS name"